lib/player.js

```diff
@@ -1020,6 +1023,18 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return this.wrapWalkerListenersWithPromise_(events);
   }
 
+  /**
+   * Can be used to update the stream startTime in the
+   * <code>manifestparsed</code> event handler using
+   * information from the manifest.
```

Comment:
I would instead say that this can be used inside events that fire during the MediaSource loading process, and call out `manifestparsed` as an example use case.
That's because there are other events you could be calling this inside. `drmsessionupdate` event, for example, could be a theoretical use-case.

Reply:
done